docs/source/Plugin/P001_commands_GPIO.repl

```diff
@@ -117,18 +117,26 @@ Supported hardware: |P000_usedby_GPIO|
   This can be useful to save energy on servos which do not need power to remain at the same position.
   "
   "
-  ``Monitor,<GPIO>``
+  ``Monitor,G,<GPIO>``

```

Comment:
What is that 'G' supposed to do?

Reply:
the command syntax is:
monitor,gpio,pin
monitor,pcf,pin
monitor,mcp,pin

that can be shrinked to:
monitor,g,pin
monitor,p,pin
monitor,m,pin